tests/rptest/scale_tests/cloud_retention_test.py

```diff
@@ -0,0 +1,119 @@
+from ducktape.utils.util import wait_until
```

Comment:
When you rebase this for the kgo-verifier merge, please can you add a license header